test/stdlib.wonton.test.ts

```diff
@@ -106,19 +106,19 @@ describe("stdlib", () => {
       it("nested array", () => {
         食渾沌(
           escape`
-物
-  之「名句」列
-    列
-      言「清平調」
-      言「雲想衣裳花相容。春風拂檻露華濃。」
-    也
-    列
-      言「長相思」
-      言「長相思。在長安。絡緯秋啼金井闌。微霜淒淒簟色寒。」
-    也
-    言「落日欲沒峴山西。倒著接蘺花下迷。襄陽小兒齊拍手。攔街爭唱「白銅鞮」。」
-  也
-也`,
+          物
+            之「名句」列
+              列
+                言「清平調」
+                言「雲想衣裳花相容。春風拂檻露華濃。」
+              也
+              列
+                言「長相思」
+                言「長相思。在長安。絡緯秋啼金井闌。微霜淒淒簟色寒。」
+              也
+              言「落日欲沒峴山西。倒著接蘺花下迷。襄陽小兒齊拍手。攔街爭唱「白銅鞮」。」
```

Comment:
Though the parser doesn't currently complain about it, maybe we don't want to support that, because wenyan is (or aims to be) typestrict and `'a arr` should only contain `'a`

Then again, we are walking further away from JSON. Perhaps if we want to support all that duck-ness, we'll need new data structures like those iterator-node-tree thingies in XML/JSON parsers in C++.

Reply:
Agree with that! Should we throw an error for it? Then I can make this test expected to see an error 😄

Reply:
Yes, ideally. Simple case is shallow type checking with numbers, bools, etc., easy. But there's some work for recursively checking objects and arrays :P